sashimi/hardware/cameras/__init__.py

```diff
@@ -0,0 +1,180 @@
+from sashimi.config import read_config
+import time
+import numpy as np
+
+conf = read_config()
+
+
+class BasicCamera:
```

Comment:
rename this to AbstractCamera because it doesn't do anything (classes that are called Abstract need the inherited ones to implement things, whereas the name Basic implies it already works in a simple case)